src/client/components/Radio/ButtonRadio.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const users = ['Student', 'Mentor', 'Admin'];
+function ButtonRadio() {
+  return (
+    <div className="btn-group">
+      {users.map((u) => (
+        <>
+          <button type = "button" name="user" className="button">
+            {' '}
+            {u}
```

Comment:
same here so you'll have **{user}** instead of **{u}**

Reply:
feedback implemented @STEP0HEN